lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -31,6 +31,118 @@
 /** The TaskService manages all operations on {@linkplain Task Tasks}. */
 public interface TaskService {
 
+  // region Task
+
+  // CREATE
+
+  /**
+   * Instantiates a non-persistent/non-inserted {@linkplain Task}.
+   *
+   * <p>Since a {@linkplain Task} doesn't allow setting a {@linkplain Task#getWorkbasketSummary()
+   * workbasketSummary}, please either provide an implementation of the {@linkplain
+   * TaskRoutingProvider} or use the referenced methods to create a {@linkplain Task} within a
+   * specific {@linkplain Workbasket}.
+   *
+   * @return the instantiated {@linkplain Task}
+   * @see #newTask(String)
+   * @see #newTask(String, String)
+   */
+  Task newTask();
```

Comment:
Can we discuss if newTask is good in the "CREATE" region, and if we need the regions?